lib/apiForSheets.js

```diff
@@ -128,6 +155,54 @@ module.exports.generateOutput = async function (rows, prNumber, prDetails) {
   }
 };
 
+module.exports.checkClaAndOnboardingTracker = async function
+(authCla) {
+  var sheets = google.sheets('v4');
+  var responseFromCla, responceFromOnBoardingSheet;
+  await sheets.spreadsheets.values.get(
+    {
+      auth: authCla,
+      spreadsheetId: spreadsheetId,
+      range: 'Usernames!A:A',
+    },
+    async function (err, response) {
+      if (err) {
+        console.log('The API returned an error: ' + err);
+        return;
+      }
+
+      responseFromCla = response;
+    }
+  );
```

Comment:
Since we are using async and await, this can be modified to look like:
https://github.com/oppia/oppiabot/blob/461207cda96c793280a026234713b117c106870b/lib/checkIssueAssignee.js#L33-L37